src/variable.js

```diff
@@ -167,6 +167,7 @@ function variable_defineImpl(name, inputs, definition) {
     this._name = name;
   }
 
+  ++this._version;
```

Comment:
This is needed when you call *variable*.define (or a wrapper like *variable*.import) so that the version is incremented synchronously. If we don’t do this, then the version won’t be implemented until the *runtime* recomputes on the next animation frame, which means we could leak a value that we already know is invalid because the variable was redefined.